packages/GoogleAnalytics/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@onaio/google-analytics",
+  "version": "0.0.0",
+  "description": "Google Analytics integration with React Router v4",
+  "main": "index.js",
+  "types": "dist/types",
+  "repository": "git+https://github.com/onaio/js-tools.git",
+  "author": "Ona Engineering",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/onaio/js-tools/issues"
+  },
+  "scripts": {
+    "jest": "jest --coverage --verbose --color",
+    "tsc": "tsc",
+    "transpile": "babel src -d dist --root-mode upward --extensions '.ts,.tsx'  --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'"
+  },
+  "files": ["dist"],
+  "publishConfig": { "access": "public" },
+  "jest": {
+    "setupFiles": ["../../setupTests"]
+  },
+  "dependencies": {
+    "@onaio/react-utils": "0.0.0",
```

Comment:
Do we need to have this as `"^0.0.0"`?

Reply:
@moshthepitt I remember reading sometime back that when the version is 0,  semver ranges don't have any impact. Is it true?

Reply:
If the package is being published in any capacity, I think it should have at least `0.0.1`, if not `0.1.0`. To me, `0.0.0` indicates....nothing.

Reply:
if this is the only thing that is left I am good with merging as is and then dealing with this when we publish.